PasswordValidator/Validators/AdvancedValidator.cs

```diff
@@ -8,6 +8,11 @@ public class AdvancedValidator : IPasswordValidator
 
         public bool Validate(string password)
         {
+            if (string.IsNullOrWhiteSpace(password))
+            {
+                throw new ArgumentException("Invlaid Null or Whitespace Password Input");
```

Comment:
Change message to $"{nameof(password)} cannot be null or whitespace" (I've done that guessing not in VS)